src/kirby/components/modal/modal-wrapper/config/modal-config.ts

```diff
@@ -1,6 +1,6 @@
 export interface ModalConfig {
   title: string;
-  titleHorizontalAlignment?: 'left' | 'center';
```

Comment:
We are using this property to set alignment of title (as on attached wireframes). Would it still be possible with the changes in this PR?
![Screenshot 2019-05-21 at 12 56 35](https://user-images.githubusercontent.com/43847833/58090618-e60f9500-7bc7-11e9-9c92-f017d19570c5.png)

Reply:
Maybe we should just have @bommariusser to comment on this. I take it he gave the instructions :)